src/service-worker.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable no-undef, no-restricted-globals */
+
+// Cache the index page
+workbox.routing.registerRoute(
+    context => context.url.pathname.indexOf('/api') !== 0 && context.url.pathname.indexOf('/assets') !== 0,
+    workbox.strategies.cacheFirst({
+        cacheName: 'pages',
+        plugins: [
+            new workbox.expiration.Plugin({
+                maxAgeSeconds: 30 * 24 * 60 * 60, // 24 hours
+            }),
```

Comment:
Do the webpack assets expire?

Reply:
To my understanding, no. The precache manifest generates revision numbers for all the webpack assets, so when we build the project with new assets then the service worker should update and update the cache.

Reply:
Yeah, trying to keep track of when the service worker updates/ when the precache update/ when the runtime cache updates is kind of confusing. I probably didn't make that any easier, but I think we're both on the same page now....

Reply:
Workbox's `injectManifest()` is kind of a hybrid between using a plugin to generate your service worker (like `Offline-plugin` or Workbox's `GenerateSW()` do) and going the complete DIY route of writing your service worker from scratch.